apps/SageAccountWeb/src/pages/MfaRequiredPage.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Button, Link, Paper, Typography } from '@mui/material'
+import { StyledOuterContainer } from '../components/StyledComponents.js'
+import mfaRequiredImageURL from '@/assets/2FAHeroImage.svg?url'
+import { useNavigate } from 'react-router'
+
+export type mfaRequiredPageProps = {}
+
+function MfaRequiredPage(props: mfaRequiredPageProps) {
+  const navigate = useNavigate()
+
+  return (
+    <StyledOuterContainer className="OAuthClientManagementPage">
```

Comment:
If this is doing any styling, please rename the class

```suggestion
    <StyledOuterContainer>
```

Reply:
Removed